pages/1.13/installing/production/system-requirements/zookeeper-requirements/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: layout.pug
+title: ZooKeeper resources
+navigationTitle: ZooKeeper resources
+menuWeight: 10
+excerpt: Requirements and recommendations for ZooKeeper in a DC/OS cluster
+---
+ZooKeeper is a centralized coordination service that stores, maintains, and synchronizes information for distributed systems. ZooKeeper and its management service maintain state information and record details of node activity in a data directory. As changes are made to the cluster, those changes are recorded in the ZooKeeper transaction log. When the transaction log grows too large, ZooKeeper creates a snapshot of the current state of cluster nodes.
+
+# Why planning ZooKeeper resources is important
+As a fundamental component of the DC/OS platform architecture, ZooKeeper performs several critical tasks for the DC/OS cluster. For example, ZooKeeper identifies which master node is used as the leader and coordinates the leader selection so that this information is available to the other master nodes, agents, and schedulers. ZooKeeper also maintains state information for Marathon-orchestrated services and persists this information in its transaction logs and snapshots.
+
+Because ZooKeeper status, operation, and performance can directly affect the stability, resiliency, and performance of the DC/OS cluster, it is important to optimize your ZooKeeper configuration to handle the intended cluster workload effectively and efficiently. For example, issues with ZooKeeper write performance often lead to latency-related problems and degraded cluster performance.
+
+# Transaction logs and snapshots
+In planning your cluster deployment, there are two important points to keep in mind for ZooKeeper:
+- ZooKeeper processes each read or write operation as a complete transaction rather than performing partial or incremental updates when there are changes to nodes in the cluster.
+- ZooKeeper does not remove old snapshots and log files from the file system. You must manage these files manually.
+
+For a DC/OS cluster, both the snapshots and transaction log are stored in the `/var/lib/dcos/exhibitor` directory. Using a dedicated log device for ZDC/OS exhibitor (`dcos-exhibitor`) log files helps to avoid resource contention and latency issues.
+
+# Identifying potential problems
```

Comment:
This is one very specific problem. I suggest removing this and adding some text below (see other comment).